service/src/main/java/com/epam/esm/service/dto/UsersCreatingDto.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.esm.service.dto;
+
+import java.util.Objects;
+
+public class UsersCreatingDto {
```

Comment:
just UserCredential

Reply:
Thank you so much for this advice. (fixed)